client/my-sites/themes/theme-options.js

```diff
@@ -3,70 +3,11 @@
  */
 import assign from 'lodash/object/assign';
 import mapValues from 'lodash/object/mapValues';
-import pick from 'lodash/object/pick';
-import merge from 'lodash/object/merge';
 
 /**
  * Internal dependencies
  */
 import Helper from 'lib/themes/helpers';
```

Comment:
The functions in this file seem a bit lonely. Maybe they could be moved to `my-sites/themes/helpers`, also I wonder if we should move `lib/themes/helpers` to the same file.